packages/x-components/src/x-modules/experience-controls/components/experience-controls.vue

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import Vue from 'vue';
+  import { XOn } from '../../../components/decorators/bus.decorators';
+  import { XEvent, XEventsTypes } from '../../../wiring/events.types';
+
+  export default class ExperienceControls extends Vue {
```

Comment:
You're missing the definition of the component with `@Component` and also registering it as an x component with `xComponentMixin(experienceControlsXModule)`

```suggestion
  import { xComponentMixin } from '../../../components/x-component.mixin';
  import { experienceControlsXModule } from '../x-module';

  @Component({
    mixins: [xComponentMixin(experienceControlsXModule)]
  })
  export default class ExperienceControls extends Vue {
```